src/app/pages/instances/stores/virtualization-instances.store.ts

```diff
@@ -36,6 +42,7 @@ export class VirtualizationInstancesStore extends ComponentStore<VirtualizationI
   }
 
   readonly initialize = this.effect((trigger$) => {
+    this.killSubscription$.next();
```

Comment:
let's use: 
`private readonly destroy$ = new Subject<void>();`

	•	“Kill” sounds harsh and doesn’t clearly indicate purpose.